### PR TITLE
app-text/cmark: New package (Bug 549682)

### PR DESCRIPTION
**Pull request by winterheart**
cmark is CommonMark parsing and rendering library and program in C.
This package is required as additional dependency for dev-games/aseprite.
Resolves bug https://bugs.gentoo.org/show_bug.cgi?id=549682.

### CODE REVIEW

File: app-text/cmark/cmark-0.27.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+
+DEPEND=""
+RDEPEND="${DEPEND}"
+
+src_prepare() {
+	default
```
Comment (SoapGentoo) [inline app-text/cmark/cmark-0.27.1.ebuild]: don't use `default` with cmake, use `cmake-utils_src_prepare` instead

**Comment by winterheart** [comment]
@SoapGentoo done

File: app-text/cmark/cmark-0.27.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+DESCRIPTION="CommonMark parsing and rendering library and program in C"
+HOMEPAGE="https://github.com/jgm/cmark"
+SRC_URI="https://github.com/jgm/cmark/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+
+DEPEND=""
```
Comment (gktrk) [inline app-text/cmark/cmark-0.27.1.ebuild]: Looks like the tests have a dependency to python3:
```
-- Found PythonInterp: /usr/bin/python3 (found suitable version "3.4.5", minimum required is "3") 
```

**Comment by winterheart** [comment]
@gktrk Added dev-lang/python to DEPEND.

**Comment by gktrk** [comment]
Merged in https://github.com/gentoo/gentoo/compare/9fda18b...57f65c1. Thanks!

**Pull request closed without merging**